### DISCUSSION TITLE
Where does this install on windows?

### DISCUSSION THREAD

**Discussion by Whiskeysaurus** [ZL-Audio/ZLEqualizer#95] [Q&A] [answered]
I've never felt so dumb... Seriously, If I just select the VST3 in the installer, where does it go?

Also for the love of god, please add the ability to choose location.

**Comment by zsliu98** [comment, marked as answer]
`C:\Program Files\Common Files\VST3`

From Steinberg

> The VST3 format has a dedicated installation path all VST3 plug-ins must comply with. All your VST3 plug-ins should be installed here.

**Reply by pyyalex** [reply]
it's not a must. 
> Any folder your VST host application is scanning during startup by default is also suitable.

would be great to have an option

**Reply by zsliu98** [reply]
> it's not a must.
> 
> > Any folder your VST host application is scanning during startup by default is also suitable.
> 
> would be great to have an option

Most DAWs do provide workarounds so that you can load VST3 from customized directories. However, as Steinberg describes it as a must, I'd better follow their rules.

**Reply by pyyalex** [reply]
fair enough
